### ISSUE TITLE
Extract pipelines from Colab notebooks

### ISSUE THREAD

**Issue by proofconstruction** [sparkfish/augraphy#150]
Each of the archetype reproduction issues* links to a Colab notebook where development occurred on the pipeline reproducing that archetype. We should take each of these pipelines and make them available as functions in the `augraphy.default` namespace, alongside the `default_augraphy_pipeline`.

* [#76 , #77 , #78 , #79 , #81 , #81 , #82 , #83 , #84 , #85 ]

**Comment by proofconstruction** [comment]
Working on this on my [colab-pipelines](https://github.com/proofconstruction/augraphy/tree/colab-pipelines) branch.

**Comment by jboarman** [comment]
@proofconstruction Can we close this issue?

**Comment by jboarman** [comment]
@kwcckw I'm reopening this issue since I believe you are still working on these.  I saw that you've extracted some of these into the available default pipelines which is great.  I'd also like to ensure we have the notebooks available in an `/examples` folder as a type of documentation that people can open to directly test and play with on their own.

**Comment by kwcckw** [comment]
> @kwcckw I'm reopening this issue since I believe you are still working on these. I saw that you've extracted some of these into the available default pipelines which is great. I'd also like to ensure we have the notebooks available in an `/examples` folder as a type of documentation that people can open to directly test and play with on their own.

Yes, i will include the notebook examples later after i completed all the archetype pipelines. Most of the time some changes will be needed in the augmentations to generate a general pipeline so I'm thinking to add those examples later once we finalized those changes.
